cfme/common/__init__.py

```diff
@@ -387,7 +389,11 @@ class EditTagsFromDetails(CFMENavigateStep):
     prerequisite = NavigateToSibling('Details')
 
     def step(self):
-        self.prerequisite_view.toolbar.policy.item_select('Edit Tags')
+        try:
```

Comment:
**OPTIOINAL:** Could you write a comment explaining under what circumstances and why you might see one widget over the other?